packages/common/src/TruffleConfig.js

```diff
@@ -37,7 +37,7 @@ const gasPx = 20000000000; // 20 gwei
 const gas = undefined; // Defining this as undefined (rather than leaving undefined) forces truffle estimate gas usage.
 
 // If a custom node URL is provided, use that. Otherwise use an infura websocket connection.
-function getNodeUrl(networkName = argv.network) {
+function getNodeUrl(networkName) {
```

Comment:
if this no longer defaults to the `argv.network` then you can properly remove the `const argv = require("minimist")(process.argv.slice(), {});` line on 17

Reply:
Nice catch! Done.